packages/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -20,6 +20,14 @@ const useStyles = makeStyles({
     padding: '12px',
     gap: '12px',
     borderRadius: theme.global.borderRadius.medium,
+
+    '> .fluentui-react-card-preview': {
+      marginLeft: '-12px',
```

Comment:
I think it is not a very good practice to use negative values for margins, maybe (if it is possible) removing the margin from the above component could be better?

Also, the values are fixed, could there be a possibility that the card might not look as it should because of this?

Reply:
Regarding negating the margins, I'm also not a fan of this solution but I think it's a compromise between elegance and utility.
I had already discussed this topic with @varholak-peter about having the padding instead on the child components (CardHeader, CardPreview, etc), which would potentially remove this issue as each component governs itself. However, I see 2 issues with this:
- If our users use Card and not the child components, the component would not follow the Fluent guidelines out of the box.
- Whenever we have the `size` prop on Card (according to the current spec), we'll need to change this padding as it won't always be the fixed 12px. This means that we'd probably need to provide some CardContext for the inner components to consume and be compliant.
Given these 2 constraints, I think the less elegant negative margin solution is the better option as it will guarantee it's Fluent by default, allow us to control the padding from the parent (whenever we have `size`) and avoid duplicate code on the inner components.
Regardless of what we choose, we felt it was a discussion that needed to happen more in depth and opted for this just to fast track it for now.

As for the mention about fixed values, 12px matches the, eventual, default `size` for card which is `medium`. Given that the `size` prop won't be implemented for phase 1, we went with a fixed value for now.